objectivec/include/ort_session.h

```diff
@@ -105,6 +105,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable instancetype)initWithError:(NSError**)error NS_SWIFT_NAME(init());
 
+/**
+ * Appends an execution provider to the session configuration options.
+ * The execution provider list is ordered by decreasing priority
```

Comment:
maybe add something like: "Available since 1.14."
in the newly added functions and classes in the public headers

Reply:
Done.

Reply:
thanks!